File: RateManagerCss/Models/OtaRequest/StatusApplicationControl.vb
Public Class StatusApplicationControl

    Private rtePlanCode As String
    Public Property RatePlanCode() As String
        Get
            Return rtePlanCode
        End Get
        Set(ByVal value As String)
            rtePlanCode = value
        End Set
    End Property

    Private roomTypeCode As String
    Public Property InvTypeCode() As String
        Get
            Return roomTypeCode
        End Get
        Set(ByVal value As String)
            roomTypeCode = value
        End Set
    End Property

End Class

File: APIServices/Conflux/Models/OTA/Restrictions/StatusApplicationControl.cs
using System;

namespace APIServices.Conflux.OTA.Models.Restrictions
{
    public class StatusApplicationControl
    {
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
        public string InvTypeCode { get; set; }
        public string RatePlanCode { get; set; }
        public bool ApplyMon { get; set; } = true;
        public bool ApplyTue { get; set; } = true;
        public bool ApplyWed { get; set; } = true;
        public bool ApplyThu { get; set; } = true;
        public bool ApplyFri { get; set; } = true;
        public bool ApplySat { get; set; } = true;
        public bool ApplySun { get; set; } = true;

    }
}
